src/modules/accounting/accounting.py

```diff
@@ -46,11 +54,11 @@ class Accounting(BaseModule, ConsensusModule):
         - Send extra data
             Contains stuck and exited validators count by each node operator.
     """
-    CONSENSUS_VERSION = 1
-    CONTRACT_VERSION = 1
+    COMPATIBLE_CONTRACT_VERSIONS = [1, 2]
```

Comment:
How about to make this more strict?

Currently all combinations of contract and consensus versions are accepted.
1,1  - correct
1,2 - unexpected AO contract upgrade
2,1 - unexpected AO contract upgrade
2,2 - correct

Maybe we should not accept unexpected combinations?

Reply:
fixed